README.md

```diff
@@ -28,11 +28,11 @@ The entries of the shared map eventually expire to not query too old data.
 
 Similarly, it starts querying `ProfileTypes` every 10 seconds to know what profile types are available.
 The result is written to a shared map.
-Every 15 seconds there are `QueryRange` requests (querying up to 15min of data) for a random profile type.
+Every 15 seconds there are `QueryRange` requests (querying 15min and 7 day ranges) for all series.
```

Comment:
just making sure, the 15 seconds is configurable since #7 right?

Reply:
Yes. I will specify. I also opened https://github.com/parca-dev/parca-load/issues/8 since this and more per-query config options would be better read from a file.